addons/mail/static/tests/discuss/core/public_web/sub_channels.test.js

```diff
@@ -327,3 +327,20 @@ test("show notification when clicking on deleted thread", async () => {
         text: "This thread is no longer available.",
     });
 });
+
+test("Can delete channel thread as author of thread", async () => {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({ name: "General" });
+    const subChannelID = pyEnv["discuss.channel"].create({
+        name: "test thread",
+        parent_channel_id: channelId,
+    });
+    await start();
+    await openDiscuss(subChannelID);
+    await contains(".o-mail-DiscussContent-threadName[title='test thread']");
+    await click(".o-mail-DiscussSidebar-item:contains('test thread') [title='Thread Actions']");
+    await click(".o-dropdown-item:contains('Delete Thread')");
+    await click(".modal button:contains('Delete Thread')");
+    await contains(".o-mail-DiscussContent-threadName[title='General']");
+    await contains(`.o-mail-NotificationMessage:contains('deleted the thread "test thread"')`);
```

Comment:
Why don't we also assert the name?

Reply:
It's best to see the full sentence, assert the actual message that is displayed rather than one of its part.